### PR TITLE
DNN-19740

### PR DESCRIPTION
**Pull request by mean2me**

### CODE REVIEW

File: src/Modules/Content/Dnn.PersonaBar.Pages/Components/PageUrlsController.cs [hunk 1]
```diff
@@ -33,6 +33,18 @@ public IEnumerable<Url> GetPageUrls(TabInfo tab, int portalId)
...
             var portalSettings = PortalController.Instance.GetCurrentPortalSettings();
+            PortalInfo aliasPortal = new PortalAliasController().GetPortalByPortalAliasID(dto.SiteAliasKey);
+
+            if (aliasPortal != null && portalSettings.PortalId != aliasPortal.PortalID)
+            {
+                    return new PageUrlResult
+                    {
+                        Success = false,
+                        ErrorMessage = Localization.GetString("CustomUrlPortalAlias.Error"),
+                        SuggestedUrlPath = ""
```
Comment (tpluscode) [inline src/Modules/Content/Dnn.PersonaBar.Pages/Components/PageUrlsController.cs]: please use `String.Empty`

File: src/Modules/Content/Dnn.PersonaBar.Pages/admin/personaBar/App_LocalResources/Pages.resx [hunk 2]
```diff
@@ -525,6 +525,9 @@
   <data name="CustomUrlPathCleaned.Error" xml:space="preserve">
     <value>The Page URL entered contains characters which cannot be used in a URL or are illegal characters for a URL.   These characters have been removed.  Click the Update button again to accept the modified URL.&lt;br&gt;[NOTE: The illegal characters list is the following: &lt;&gt;\?:&amp;=+|%# ]</value>
   </data>
+  <data name="CustomUrlPortalAlias.Error" xml:space="preserve">
+    <value>Please, select a valid site alias for this custom URL.</value>
```
Comment (tpluscode) [inline src/Modules/Content/Dnn.PersonaBar.Pages/admin/personaBar/App_LocalResources/Pages.resx]: There should no comma following **Please** 🎩 🇬🇧

**Review by tpluscode — CHANGES_REQUESTED** [review]

File: src/Modules/Content/Dnn.PersonaBar.Pages/admin/personaBar/App_LocalResources/Pages.resx (hunk 2 above)
Comment (mean2me) [reply src/Modules/Content/Dnn.PersonaBar.Pages/admin/personaBar/App_LocalResources/Pages.resx]: typo

**Review by tpluscode — APPROVED** [review]

**Comment by tpluscode** [comment]
fixes #561

**Pull request merged**